src/components/Input.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Input({ 
+    type, 
+    placeHolder
+}) {
```

Comment:
Só quebra em linhas se for muitos parametros
```suggestion
export default function Input({   type, placeHolder }) {
```

Reply:
pronto